examples-ml/pom-standalone.xml

```diff
@@ -163,6 +157,60 @@
         </dependency>
     </dependencies>
 
+    <profiles>
+        <profile>
+            <id>scala</id>
```

Comment:
Why spark profile is called `scala` but spark 2.4 profile is called `spark-2.4'? I assume scala should include both.

Reply:
According to naming, it's not my desition. I took it from Ignite. As the next change, I will move spark-2.3 to separate modules. It was discussed with Stan Lukyanov (as you remember it was done in first variant of PR but it also means the changes that differ from Ignite codebase).

Scala can't use both of them because this module will be used for example. Examples at the moment can use only one version of the spark. If I added them both to scala then they will not be able to build.  Otherwise, I can change this behavior but it also will mean the changes that differ from Ignite codebase). It should be done in Ignite first.